src/h5web/vis-packs/core/utils.ts

```diff
@@ -100,13 +100,16 @@ export function getBounds(
 
   return values.reduce<Bounds>(
     (acc, val, i) => {
+      if (!Number.isFinite(val)) {
+        return acc;
```

Comment:
Also, maybe just add a comment here to say that this ignores `Infinity` and `NaN` values from the domain computation.